### ISSUE TITLE
Prebuilt version broken

### ISSUE THREAD

**Issue by thepixelmonk** [akrennmair/libmp3lame-js#9]
When trying to load the prebuilt version I'm getting this.. any help?

```
uncaught exception: abort() at qd@http://localhost:3000/assets/libmp3lame.min.js?body=1:223
O@http://localhost:3000/assets/libmp3lame.min.js?body=1:1
Qi@http://localhost:3000/assets/libmp3lame.min.js?body=1:1
Lame</j.cwrap@http://localhost:3000/assets/libmp3lame.min.js?body=1:236
Lame<@http://localhost:3000/assets/libmp3lame.min.js?body=1:580
@http://localhost:3000/assets/libmp3lame.min.js?body=1:1

"Assertion failed: Cannot call unknown function get_lame_version (perhaps LLVM optimizations or closure removed it?)"
```

**Comment by tokudu** [comment]
I'm seeing the same issue. Also:

```
Uncaught abort() at Error
    at Error (native)
    at qd (http://localhost:5000/js/libmp3lame.min.js:223:143)
```

**Comment by kobigurk** [comment]
take a look at my pull request, it fixes it amongst other things

**Comment by xkraty** [comment]
@kobigurk could you kindly provide a compiled version of your fork? According to brew page ( https://github.com/Homebrew/homebrew/issues/29988 ) emscripten formula still broken on yosemite and i can't get it to work by manually installing it.

Thanks in advance
Chris

**Comment by kobigurk** [comment]
Hi Chris,

Take a look at my fork, it has both the compiled version and a minified one.
https://github.com/kobigurk/libmp3lame-js

**Comment by xkraty** [comment]
I already checked your dist folder but it looks like they are still the same as the original version. Or I'm wrong?

**Comment by kobigurk** [comment]
You're correct, I mistakenly didn't commit the new compiled files. They're pushed now, thanks, take a look.

**Comment by xkraty** [comment]
Awesome! Thanks, pulled and now it works properly with recording longer than 1 minute! Really thanks Kobi I owe you a beer!!

**Comment by kobigurk** [comment]
No problem, have fun :-)

**Comment by malstrom72** [comment]
Yes, the prebuilt minified version is bad. I am no emscripten expert, but I got it working by "exporting" all wrapped c functions like this: Module["_get_lame_version"] = _get_lame_version; etc and then ran it through closure myself. Got the code size down to 512kb which isn't too shabby.

**Comment by stalane** [comment]
@malstrom72 Can you share the minified version?
